.github/workflows/node.js.yml

```diff
@@ -25,7 +25,8 @@ jobs:
       - run: yarn --frozen-lockfile
       - run: |
           echo ::add-matcher::.github/eslint.json
-          yarn run eslint . --format=compact
+          yarn run eslint src test --format=compact
```

Comment:
I think the `eslint .` works in CI because we haven’t generated any files yet (whereas it will fail locally with errors on generated files in build and types). So, this is just being more explicit about what we’re testing so that it’s not confusing when you try to run this command locally.